test/grouping.jl

```diff
@@ -1332,6 +1363,7 @@ end
     @test length(gdkeys) == length(expected)
     @test size(gdkeys) == size(expected)
     @test eltype(gdkeys) == DataFrames.GroupKey{typeof(gd)}
+    @test IndexStyle(gdkeys) == IndexLinear()
```

Comment:
this is already tested four lines above

Reply:
Not sure why I thought I hadn't put that in yet.

Reply:
@nalimilan did 😄